### PR TITLE
Support attribution for reality models (OSM Building)

### PR DESCRIPTION
**Pull request by RBBentley**
Generalize the methods for providing attribution logo cards to include any reality model.  Add property to (context) reality models to store attribution properties and set this for the OpenStreetMap building reality model.

### CODE REVIEW

File: core/common/src/DisplayStyleSettings.ts [hunk 1]
```diff
@@ -78,6 +78,13 @@ export interface OrbitGtBlobProps {
   accountName: string;
 }
 
+/** JSON representation of the attribution properties for a reality model. These are used to create the logo card for the model using [IModelApp.makeLogoCard]($frontend).
+ * @beta
+ */
+export interface RealityModelAttribution {
+  heading: string;
```
Comment (pmconne) [inline core/common/src/DisplayStyleSettings.ts:85]: Please describe how each gets displayed by the logo card, or link to `makeLogoCard`.

File: core/frontend/src/DisplayStyleState.ts [hunk 2]
```diff
@@ -250,7 +250,7 @@ export abstract class DisplayStyleState extends ElementState implements DisplayS
       const tilesetUrl = getCesiumOSMBuildingsUrl();
       const name = IModelApp.i18n.translate("iModelJs:RealityModelNames.OSMBuildings");
       currentIndex = this._contextRealityModels.length;
-      this.attachRealityModel({ tilesetUrl, name, planarClipMask: { mode: PlanarClipMaskMode.None } });
+      this.attachRealityModel({ tilesetUrl, name, planarClipMask: { mode: PlanarClipMaskMode.None }, attribution: { heading: "OpenStreetMap", notice: "&copy;<a href=\"https://www.openstreetmap.org/copyright\">OpenStreetMap</a> contributors" }});
```
Comment (pmconne) [inline core/frontend/src/DisplayStyleState.ts:253]: Localization? ("contributors").

File: core/frontend/src/tile/TileTreeReference.ts [hunk 3]
```diff
@@ -206,4 +207,7 @@ export abstract class TileTreeReference /* implements RenderMemory.Consumer */ {
 
   /**  Return the clip mask priority for this model - models will be clipped by any other viewed model with a higher proirity.  BIM models have highest prioirty and are never clipped */
   public get planarClipMaskPrority(): number { return PlanarClipMaskPriority.BIM; }
+
+  /** Add attribution logo cards for the tile tree source logo cards to logo div. */
```
Comment (pmconne) [inline core/frontend/src/tile/TileTreeReference.ts:211]: Add `@see [[IModelApp.makeLogoCard)]]`.

File: core/common/src/DisplayStyleSettings.ts [hunk 4]
```diff
@@ -78,6 +78,13 @@ export interface OrbitGtBlobProps {
   accountName: string;
 }
 
+/** JSON representation of the attribution properties for a reality model. These are used to create the logo card for the model using [IModelApp.makeLogoCard]($frontend).
+ * @beta
+ */
+export interface RealityModelAttribution {
```
Comment (pmconne) [inline core/common/src/DisplayStyleSettings.ts:84]: Might some people want to supply an icon as supported by `makeLogoCard`? Perhaps this and the argument to `makeLogoCard` should simply be called `LogoCardProps`?

Comment (RBBentley) [reply core/common/src/DisplayStyleSettings.ts:84]: Yes - but Is there a way to store an icon as JSON?

Comment (pmconne) [reply core/common/src/DisplayStyleSettings.ts:84]: Only if it's a string that refers to a resource bundled with the app - so not usefully, no.

**Pull request closed without merging**